setup.py

```diff
@@ -36,7 +36,7 @@ def read(filename):
       zip_safe=False,
       install_requires=[
           'setuptools',
-          # -*- Extra requirements: -*-
+          'colorama',
```

Comment:
So... this is where we cross the line towards dependencies :-)

I've long tried to keep zest.releaser working without dependencies, but I guess now is the time to do it. With wheels and so... And there was some new generic library that might be handy to upload stuff to pypi. Etc.